app/views/docs/databases.phtml

```diff
@@ -33,18 +33,13 @@
 
 <h3><a href="/docs/databases#permissions" id="permissions">Permissions</a></h3>
 <p>
-    Once you create your collection, you will reach your collection's settings page, where you can choose the permission model for your collection. There are two types of permissions models available, each one with different pros and cons designed to be flexible to best match your use case.
+    Appwrite provides permissions to restrict access to documents at two levels, document and collection level. When a user has the appropriate type of <a href="/docs/permissions/">access permissions</a> granted at **either** the document or the collection level, they will be able to access or change the document. If the permission field is left empty, no one can access the document.
```

Comment:
> If the permission field is left empty, no one can access the document.

This is not entirely true since Server SDKs can access it... Can we rephrase the last sentence ?